Rest.Services/Repositories/ProductRepository.cs

```diff
@@ -0,0 +1,37 @@
+using Rest.Models;
+namespace Rest.Services;
+public class ProductRepository : IRepository<long, Product>
+{
+    public Dictionary<long, Product> _productDictionary;
```

Comment:
The underscore `_` sing infers that your field, method, etc., is private. Please verify if you want this field to be `private` or `public`.

Reply:
Understood. It is intended to be a private field. I just forgot to correct it after I did some tests with xUnit for unit testing. I'll change the access modifier.

Reply:
If for some reason you need to access to this private resource, create a public method for it.